### DISCUSSION TITLE
Is "GlobalParams" are supported?

### DISCUSSION THREAD

**Discussion by dorMelamedOolo** [microsoft/RulesEngine#189] [Q&A] [answered]
Hi,

On the Wiki page, I saw that "LocalParams" are supported. But nothing about "GlobalParams".

On the https://microsoft.github.io/RulesEngine/ page, I saw an example for a use of the "GlobalParams". So I tried to use it and failed.

Is the "GlobalParams" feature is working? maybe I'm doing something wrong?

this is the example from - https://microsoft.github.io/RulesEngine/ :

{
  "WorkflowName": "workflowWithGlobalParam",
  "GlobalParams":[
    {
      "Name":"myglobal1",
      "Expression":"myInput.hello.ToLower()"
    }
  ],
  "Rules":[
    {
      "RuleName": "checkGlobalEqualsHello",
      "Expression":"myglobal1 == \"hello\""
    },
    {
      "RuleName": "checkGlobalEqualsInputHello",
      "Expression":"myInput.hello.ToLower() == myglobal1"
    }
  ]
}

**Comment by abbasc52** [comment, marked as answer]
Global params have been added as a part of RulesEngine v3.2.0

Here is a working example based on the page :-  https://dotnetfiddle.net/aiBGst
